reya_actions/utils/execute_core_commands.py

```diff
@@ -0,0 +1,9 @@
+def execute_core_commands(config, account_id: int, commands: list):
+    w3 = config['w3']
+    account = config['w3account']
+    core = config['w3contracts']['core']
+        
+    tx_hash = core.functions.execute(account_id, commands).transact({'from': account.address})
```

Comment:
question applicable to all contract calls? what if these txs fail? will it print the decoded error?